docker/openemr/5.0.2/openemr.conf

```diff
@@ -1,71 +1,82 @@
+Header unset ETag
+# Set HSTS and X-XSS protection
+Header set Strict-Transport-Security "max-age=31536000; includeSubDomains; preload"
+Header set X-XSS-Protection "1; mode=block"
+ServerAdmin webmaster@localhost
+# Narrow document root
+DocumentRoot /var/www/localhost/htdocs/openemr
+ErrorLog ${APACHE_LOG_DIR}/error.log
+CustomLog ${APACHE_LOG_DIR}/access.log combined
 
-</VirtualHost>
-        <VirtualHost _default_:443>
 
-                #   SSL Engine Switch:
-                #   Enable/Disable SSL for this virtual host.
-                SSLEngine on
-				
-				SSLCipherSuite HIGH:!MEDIUM:!aNULL:!MD5:!RC4
-				SSLProtocol -ALL +TLSv1.2
-                #   SSLCertificateFile directive is needed.
-                SSLCertificateFile      /etc/ssl/certs/webserver.cert.pem
-                SSLCertificateKeyFile /etc/ssl/private/webserver.key.pem
+    <Directory /var/www/localhost/htdocs/openemr>
+        # Only allow these HTTP Methods
+        AllowMethods GET POST HEAD OPTIONS
+        # No indexes anywhere
+        Options -Indexes
+        AllowOverride FileInfo
+        Require all granted
+    </Directory>
+
+     <Directory "/var/www/localhost/htdocs/openemr/sites">
+         AllowOverride None
+     </Directory>
 
-                #SSLOptions +FakeBasicAuth +ExportCertData +StrictRequire
-                <FilesMatch "\.(cgi|shtml|phtml|php)$">
-                                SSLOptions +StdEnvVars
-                </FilesMatch>
-                <Directory /usr/lib/cgi-bin>
-                                SSLOptions +StdEnvVars
-                </Directory>
+     <Directory "/var/www/localhost/htdocs/openemr/sites/*/documents">
+         Require all denied
+     </Directory>
 
-        </VirtualHost>
+     <Directory "/var/www/localhost/htdocs/openemr/sites/*/edi">
+         Require all denied
+     </Directory>
 
-# vim: syntax=apache ts=4 sw=4 sts=4 sr noet
\ No newline at end of file
+     <Directory "/var/www/localhost/htdocs/openemr/sites/*/era">
+         Require all denied
+     </Directory>
+
+
+#######################################
+### Uncomment the following 3 lines ###
+### to enable HTTPS redirection #######
+### and require HTTPS only ############
+#######################################
+<VirtualHost *:80>
+        #RewriteEngine On
+        #RewriteCond %{HTTPS} off
+        #RewriteRule (.*) https://%{HTTP_HOST}/$1 [R,L]
+</VirtualHost>
+
+
+<VirtualHost _default_:443>
+        #   SSL Engine Switch:
+        #   Enable/Disable SSL for this virtual host.
+        SSLEngine on
+
+        SSLCipherSuite "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 \
+                       EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 \
+                       EECDH EDH+aRSA RC4 !aNULL !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS"
+        SSLProtocol -ALL +TLSv1.2
+        SSLCertificateFile      /etc/ssl/certs/selfsigned.cert.pem
+        SSLCertificateKeyFile /etc/ssl/private/selfsigned.key.pem
```

Comment:
@danehrlich1 ,
Doesn't this break the way the certs are set up in the docker?
-brady